### PR TITLE
Bootstrap for frontend

### PR DESCRIPTION
**Pull request by nathf**
* Basic setup of frontend react code base
* Fleshed out most of the webpack config
* Test suite started.
* Got initial component rendering through ruby
* Added content to READMEs

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
     "babel-loader": "^6.2.4",
-    "babel-runtime": "^6.6.1",
     "babel-polyfill": "^6.7.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "babel-runtime": "^6.11.6",
     "es5-shim": "^4.5.7",
     "expose-loader": "^0.7.1",
+    "gov-au-ui-kit": "git+https://github.com/nathf/gov-au-ui-kit.git#feature/node-install-issues",
```
Comment (nathf) [inline client/package.json]: This ref is only temporary until some node install issues are resolved on the ui-kit side

**Comment by micapam** [comment]
LGTM!

File: client/app/constants.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import mirror from 'mirror-creator'
```
Comment (alecky) [inline client/app/constants.js]: Do we really need this convenience method?

File: client/app/containers/Landing.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import LandingWidget from '../components/LandingWidget'
```
Comment (alecky) [inline client/app/containers/Landing.js]: rename this

File: client/app/reducers/landing.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// import actionTypes from '../constants'
+const initialState = {
+  name: ''
+}
+export default function landing (state = initialState, action) {
```
Comment (alecky) [inline client/app/reducers/landing.js]: try to inline the initial state

**Comment by alecky** [comment]
Im fine with these changes

**Pull request merged**